webpack.config.js

```diff
@@ -0,0 +1,97 @@
+const path = require('path'),
+  webpack = require('webpack'),
+  CleanWebpackPlugin = require('clean-webpack-plugin'),
+  HtmlWebpackPlugin = require('html-webpack-plugin'),
```

Comment:
Should it be declared with global scope? What is the purpose of that? Have you forgotten about const/let declarations?